test/wrap_test.rb

```diff
@@ -76,10 +76,15 @@ class BandDecorator < Representable::Decorator
 
   it do
     band.from_hash({"bands" => {"name"=>"Social Distortion"}}).name.must_equal "Social Distortion"
-    band.from_hash({"name"=>"Social Distortion"}, wrap: false).name.must_equal "Social Distortion"
-    band.from_hash({band: {"name"=>"Social Distortion"}}, wrap: :band).name.must_equal "Social Distortion"
+    band.from_hash({"name"=>"Bad Religion"}, wrap: false).name.must_equal "Bad Religion"
+    band.from_hash({"band"=>{"name"=>"Pennywise"}}, wrap: :band).name.must_equal "Pennywise"
```

Comment:
@apotonick as long as `filter_wrap_for` calls `#to_s` on `wrap` (and we don't mix in `AllowSymbols`), this is necessary.